jms-client-source/src/main/java/no/vegvesen/ixn/JmsSource.java

```diff
@@ -33,7 +33,8 @@ public void run(String... args) throws Exception {
         SSLContext sslContext = SSLContextFactory.sslContextFromKeyAndTrustStores(keystoreDetails, trustStoreDetails);
         try(Source s = new Source(properties.getUrl(),properties.getSendQueue(),sslContext)) {
             s.start();
-            s.send("Yo!", "NO", ",someukquadtile");
+            s.send("Dette er en test, FISK!", "NO", ",01230122");
+            s.sendDenmByteMessage("Dette er en test, FISK!", "NO", ",01230122");
```

Comment:
please separate the sending of text messages and denm byte message. Only send one message in the source at a time.

Reply:
Done :)